interchange/DeviceResources.capnp

```diff
@@ -213,12 +233,18 @@ struct Device {
   struct SitePIP {
     inpin  @0 : BELPinIdx $belPinRef();
     outpin @1 : BELPinIdx $belPinRef();
+    # Interconnect delay
+    delay  @2 : CornerModel;
```

Comment:
I think there was a case in the 7-series where site pip delay varied slightly on fanout. Unlikely to be important enough for a first pass but worth thinking about in the future.